airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1114,18 +1184,18 @@ def get_schema(self, dataset_id, table_id):
         :param table_id: the table ID of the requested table
         :return: a table schema
         """
-        tables_resource = self.service.tables() \
-            .get(projectId=self.project_id, datasetId=dataset_id, tableId=table_id) \
```

Comment:
I prefer this, it looks more cleaner

Reply:
just saw what, max-line-length = 90 in Airflow.....
removed changes with line 79

I will be more clever next time